### PR TITLE
Update docs for 0.9.0

### PR DESCRIPTION
**Pull request by taylorfturner**
Docs build on fork: https://taylorfturner.github.io/DataProfiler/docs/0.9.0/html/index.html

### CODE REVIEW

File: docs/source/dataprofiler.profilers.rst [hunk 1]
```diff
@@ -22,6 +22,7 @@ Modules
    dataprofiler.profilers.graph_profiler
    dataprofiler.profilers.histogram_utils
    dataprofiler.profilers.int_column_profile
+   dataprofiler.profilers.json_decoder
```
Comment (JGSweets) [inline docs/source/dataprofiler.profilers.rst:25]: do we want the decoder stuff? It's not completely functional right now. Thoughts?

Comment (ksneab7) [reply docs/source/dataprofiler.profilers.rst:25]: I think that might be next release. Just because we have a decent amount work left on it

Comment (taylorfturner) [reply docs/source/dataprofiler.profilers.rst:25]: Agree

Comment (taylorfturner) [reply docs/source/dataprofiler.profilers.rst:25]: will fix

Comment (taylorfturner) [reply docs/source/dataprofiler.profilers.rst:25]: removing decoder

**Comment by taylorfturner** [comment]
I have looked at the security check related to `tornado-6.2-cp37-abi3-` but when I install `requirements.txt` on `gh-pages` it does in fact install `6.3.2` when I look at `pip list | grep -i tornado`

**Pull request merged**